src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowTriggerRelease.java

```diff
@@ -150,30 +146,17 @@ public void perform(
       run.addAction(action);
       run.save();
       logger.println("TriggerRelease  result: " + formatJsonOutput(releaseResult));
-    } catch (IOException | InterruptedException e) {
```

Comment:
@justnoxx handling of these both exceptions seems to be similar, so we can use pipe here